src/pages/Authentication.js

```diff
@@ -0,0 +1,48 @@
+import React, { memo, useState } from 'react';
+import { Redirect, useRouteMatch, Link } from 'react-router-dom';
+
+import { useDispatch, useSession, useStatus } from 'hooks';
+import AuthenticationForm from 'components/user/AuthenticationForm';
```

Comment:
hey @juanfopaez I'm not able to find the location for this file, did you miss to commit it?

Reply:
✅